src/components/controlls/Card/Card.scss

```diff
@@ -0,0 +1,73 @@
+.card {
+  display: inline-block;
+  margin-left: 15%;
+
+  width: 80vw;
+
+  background: #ffffff;
+  border: 2px solid rgba(45, 97, 232, 0.17);
+  border-radius: 4px;
+
+  h2 {
+    font-family: 'PT Root UI';
+    display: inline-block;
+    font-size: 24px;
+    line-height: 30px;
+    overflow: hidden;
+    margin: 20px 10vw;
```

Comment:
It does not require the use of vw. In this situation, I'll go perfectly with fixed values.